### DISCUSSION TITLE
Handling bad variable identifiers

### DISCUSSION THREAD

**Discussion by Sagebati** [mitsuhiko/minijinja#480] [Q&A] [answered]
Hi, 

Little question, in our app we handle a lot of "dynamic" templates supplied by users, sometimes they use invalid characters as vairables `{{ é }}`. With this kind of input the render crashes, is there a way to handle this kind of error ?  

Many thanks for your work and cool library.

**Comment by mitsuhiko** [comment, marked as answer]
In this particular case you can enable the `unicode` feature which makes such valid variable names valid.  And yes, you can normally handle that error by matching on the return value and inspect the `Err(..)`.
